requests/cookies.py

```diff
@@ -393,8 +394,8 @@ def morsel_to_cookie(morsel):
         expires = time.time() + morsel['max-age']
     elif morsel['expires']:
         time_template = '%a, %d-%b-%Y %H:%M:%S GMT'
```

Comment:
The honest answer is "probably not", to be honest: the RFC really doesn't require it at all, and instead only specifies that the time is to be parsed as UTC.